frontend/src/routes/Infrastructure/Clusters/ClusterPools/ClusterPools.test.tsx

```diff
@@ -172,11 +235,21 @@ describe('ClusterPools page', () => {
 
     test('should be able to claim a cluster', async () => {
         await waitForText(mockClusterPool.metadata.name!)
-        await clickByText('Claim cluster')
+        await clickByText('Claim cluster', 0)
         await waitForText('Cluster claim name')
         await typeByTestId('clusterClaimName', mockClusterClaim.metadata.name!)
         const createNocks: Scope[] = [nockCreate(mockClusterClaim), nockGet(mockClusterClaim)]
         await clickByText('Claim')
         await waitForNocks(createNocks)
     })
+
+    test('should be able to claim a cluster with only standby clusters', async () => {
```

Comment:
@KevinFCormier Added a test to confirm ClusterPools with only standby clusters can be claimed. Required me to create a second mock cluster pool with 0 ready clusters. Let me know if this looks good.